### DISCUSSION TITLE
Use as web components error: Vuestic cache plugin is not registered!

### DISCUSSION THREAD

**Discussion by beholdr** [epicmaxco/vuestic-ui#3915] [Q&A] [answered]
Hi, guys!

I want to use vuestic as web components and can't figure out how to do it. I always get error in console "Vuestic cache plugin is not registered!"

Could you please check my example in the [Sandbox](https://codesandbox.io/s/cranky-water-ttzkg7) and correct me, what am I'm doing wrong?

**Comment by m0ksem** [comment, marked as answer]
Hey, @beholdr. We had a regression after updating vue compiler. Please, check if your issue is resolved in version `0.0.0-experimental-87ff25dfb-20230927`.

**Reply by beholdr** [reply]
Yeah, this version works!
